packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/mentionIntegration.tests.js

```diff
@@ -641,5 +641,28 @@ export default function() {
             this.$element.find('p').first().text('@');
             this.clock.tick(10);
         });
+
+        test('aria-activedescendant should exist on textbox when mention pops up', function(assert) {
+            const done = assert.async();
+            const valueChangeSpy = sinon.spy(() => {
+                if(valueChangeSpy.calledOnce) {
+                    this.clock.tick(10);
+                    const $items = this.getItems();
+                    const $focusedItem = $items.filter(`.${FOCUSED_STATE_CLASS}`).first();
+                    const $content = this.$element.find(`.${HTML_EDITOR_CONTENT}`);
+
+                    assert.ok($content.attr('aria-activedescendant'), 'textbox should have aria-activedescendant');
```

Comment:
Checking for the presence of an attribute does not in itself carry important information. We have to compare the values. You do this in the next line, so the current line is unnecessary.